### ISSUE TITLE
Random Javascript error dialogs showing up

### ISSUE THREAD

**Issue by edhans** [streetwriters/notesnook#3293]
### What happened?

I am randomly getting this on my Windows 11 machine. Just started after the upgrade. It just pops up once in a while so it must be something in the background.

<img width="405" alt="image" src="https://github.com/streetwriters/notesnook/assets/45298358/c0a1672c-ffb1-48e5-aa83-6795a1a01de6">

Clicking OK works and client doesn't crash. Just annoying. All of today's logs are below.

### Steps to reproduce the problem

Nothing. Just open the client and let it alone. 

### Version

2.6.2

### Platform/OS

Windows

### Relevant log output

```shell
{"level":3,"message":"Getting route for path: /","timestamp":1694189121658,"scope":"notesnook-web","elapsed":1}
{"level":3,"message":"Access token requested","timestamp":1694189124141,"extras":{"accessToken":"240AE6F909"},"scope":"TokenManager","elapsed":2484}
{"level":3,"message":"Database initialization took 2483.90ms","timestamp":1694189124142,"scope":"notesnook-web","elapsed":2484}
{"level":3,"message":"Access token requested","timestamp":1694189124157,"extras":{"accessToken":"240AE6F909"},"scope":"TokenManager","elapsed":15}
{"level":3,"message":"app render took 106.50ms","timestamp":1694189124302,"scope":"notesnook-web","elapsed":160}
{"level":3,"message":"Access token requested","timestamp":1694189125551,"extras":{"accessToken":"240AE6F909"},"scope":"TokenManager","elapsed":1394}
{"level":3,"message":"Access token requested","timestamp":1694189125600,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":0}
{"level":3,"message":"Access token requested","timestamp":1694189125606,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":0}
{"level":3,"message":"Access token requested","timestamp":1694189125648,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":42}
{"level":3,"message":"Sync status updated: syncing","timestamp":1694189126113,"scope":"notesnook-web","elapsed":1811}
{"level":3,"message":"Auto sync started","timestamp":1694189126119,"scope":"AutoSync","elapsed":0}
{"level":3,"message":"Starting sync","timestamp":1694189126149,"extras":{"full":true,"force":false,"serverLastSynced":null},"scope":"Sync","elapsed":4492}
{"level":3,"message":"Access token requested","timestamp":1694189126187,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":4530}
{"level":3,"message":"Ignoring duplicate sync","timestamp":1694189126434,"extras":{"full":true,"force":false,"lastSynced":null,"syncing":true,"syncDisabled":false,"offline":false},"scope":"notesnook-web","elapsed":321}
{"level":3,"message":"Access token requested","timestamp":1694189126473,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":825}
{"level":3,"message":"Initialized sync","timestamp":1694189127014,"extras":{"lastSynced":1694181965714,"oldLastSynced":1694181965714},"scope":"Sync","elapsed":865}
{"level":3,"message":"Data fetched","timestamp":1694189127342,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":328}
{"level":3,"message":"Uploading attachments...","timestamp":1694189127344,"extras":{"total":0},"scope":"Sync","elapsed":2}
{"level":3,"message":"Stopping sync","timestamp":1694189130554,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":0}
{"level":3,"message":"Sync status updated: completed","timestamp":1694189130588,"scope":"notesnook-web","elapsed":4154}
{"level":3,"message":"Access token requested","timestamp":1694189130622,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":4149}
{"level":3,"message":"refreshing app took 587.80ms","timestamp":1694189131178,"scope":"notesnook-web","elapsed":590}
{"level":3,"message":"Sync status updated: completed","timestamp":1694189131222,"scope":"notesnook-web","elapsed":44}
{"level":3,"message":"Sync status updated: synced","timestamp":1694189133593,"scope":"notesnook-web","elapsed":2371}
{"level":3,"message":"Sync status updated: syncing","timestamp":1694189137725,"scope":"notesnook-web","elapsed":4132}
{"level":3,"message":"Auto sync requested","timestamp":1694189137726,"scope":"AutoSync","elapsed":11607}
{"level":3,"message":"Starting sync","timestamp":1694189137734,"extras":{"full":true,"force":false,"serverLastSynced":null},"scope":"Sync","elapsed":7180}
{"level":3,"message":"Initialized sync","timestamp":1694189137781,"extras":{"lastSynced":1694181965714,"oldLastSynced":1694181965714},"scope":"Sync","elapsed":47}
{"level":3,"message":"Data fetched","timestamp":1694189138119,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":338}
{"level":3,"message":"Uploading attachments...","timestamp":1694189138120,"extras":{"total":0},"scope":"Sync","elapsed":1}
{"level":3,"message":"Stopping sync","timestamp":1694189141714,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":0}
{"level":3,"message":"Sync status updated: completed","timestamp":1694189141772,"scope":"notesnook-web","elapsed":4047}
{"level":3,"message":"Access token requested","timestamp":1694189141825,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":11203}
{"level":3,"message":"refreshing app took 593.80ms","timestamp":1694189142368,"scope":"notesnook-web","elapsed":596}
{"level":3,"message":"Sync status updated: completed","timestamp":1694189142460,"scope":"notesnook-web","elapsed":92}
{"level":3,"message":"Sync status updated: synced","timestamp":1694189144778,"scope":"notesnook-web","elapsed":2318}
{"level":3,"message":"Page visibility changed. Reconnecting SSE...","timestamp":1694190031576,"scope":"notesnook-web","elapsed":886798}
{"level":3,"message":"Sync status updated: syncing","timestamp":1694190032206,"scope":"notesnook-web","elapsed":630}
{"level":3,"message":"Auto sync requested","timestamp":1694190032208,"scope":"AutoSync","elapsed":894482}
{"level":3,"message":"Starting sync","timestamp":1694190032214,"extras":{"full":true,"force":false,"serverLastSynced":null},"scope":"Sync","elapsed":890500}
{"level":3,"message":"Initialized sync","timestamp":1694190032248,"extras":{"lastSynced":1694181965714,"oldLastSynced":1694181965714},"scope":"Sync","elapsed":34}
{"level":3,"message":"Data fetched","timestamp":1694190032954,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":706}
{"level":3,"message":"Uploading attachments...","timestamp":1694190032955,"extras":{"total":0},"scope":"Sync","elapsed":1}
{"level":3,"message":"Stopping sync","timestamp":1694190035139,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":0}
{"level":3,"message":"Sync status updated: completed","timestamp":1694190035163,"scope":"notesnook-web","elapsed":2957}
{"level":3,"message":"Access token requested","timestamp":1694190035188,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":893363}
{"level":3,"message":"refreshing app took 565.60ms","timestamp":1694190035731,"scope":"notesnook-web","elapsed":568}
{"level":3,"message":"Sync status updated: completed","timestamp":1694190035771,"scope":"notesnook-web","elapsed":40}
{"level":3,"message":"Sync status updated: synced","timestamp":1694190038504,"scope":"notesnook-web","elapsed":2733}
{"level":3,"message":"Access token requested","timestamp":1694190070857,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":949200}
{"level":3,"message":"Page visibility changed. Reconnecting SSE...","timestamp":1694190118890,"scope":"notesnook-web","elapsed":80386}
{"level":3,"message":"Sync status updated: syncing","timestamp":1694190119462,"scope":"notesnook-web","elapsed":572}
{"level":3,"message":"Auto sync requested","timestamp":1694190119464,"scope":"AutoSync","elapsed":87256}
{"level":3,"message":"Starting sync","timestamp":1694190119470,"extras":{"full":true,"force":false,"serverLastSynced":null},"scope":"Sync","elapsed":84331}
{"level":3,"message":"Initialized sync","timestamp":1694190119508,"extras":{"lastSynced":1694181965714,"oldLastSynced":1694181965714},"scope":"Sync","elapsed":38}
{"level":3,"message":"Data fetched","timestamp":1694190119797,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":289}
{"level":3,"message":"Uploading attachments...","timestamp":1694190119798,"extras":{"total":0},"scope":"Sync","elapsed":1}
{"level":3,"message":"Stopping sync","timestamp":1694190121625,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":0}
{"level":3,"message":"Sync status updated: completed","timestamp":1694190121636,"scope":"notesnook-web","elapsed":2174}
{"level":3,"message":"Access token requested","timestamp":1694190121662,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":86474}
{"level":3,"message":"refreshing app took 991.60ms","timestamp":1694190122629,"scope":"notesnook-web","elapsed":993}
{"level":3,"message":"Sync status updated: completed","timestamp":1694190123258,"scope":"notesnook-web","elapsed":629}
{"level":3,"message":"Sync status updated: synced","timestamp":1694190124641,"scope":"notesnook-web","elapsed":1383}
{"level":3,"message":"Page visibility changed. Reconnecting SSE...","timestamp":1694190168431,"scope":"notesnook-web","elapsed":43790}
{"level":3,"message":"Sync status updated: syncing","timestamp":1694190168973,"scope":"notesnook-web","elapsed":542}
{"level":3,"message":"Auto sync requested","timestamp":1694190168977,"scope":"AutoSync","elapsed":49513}
{"level":3,"message":"Starting sync","timestamp":1694190168982,"extras":{"full":true,"force":false,"serverLastSynced":null},"scope":"Sync","elapsed":47357}
{"level":3,"message":"Initialized sync","timestamp":1694190169022,"extras":{"lastSynced":1694181965714,"oldLastSynced":1694181965714},"scope":"Sync","elapsed":40}
{"level":3,"message":"Data fetched","timestamp":1694190169322,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":300}
{"level":3,"message":"Uploading attachments...","timestamp":1694190169323,"extras":{"total":0},"scope":"Sync","elapsed":1}
{"level":3,"message":"Stopping sync","timestamp":1694190171418,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":0}
{"level":3,"message":"Sync status updated: completed","timestamp":1694190171429,"scope":"notesnook-web","elapsed":2456}
{"level":3,"message":"Access token requested","timestamp":1694190171456,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":49794}
{"level":3,"message":"refreshing app took 978.60ms","timestamp":1694190172412,"scope":"notesnook-web","elapsed":983}
{"level":3,"message":"Sync status updated: completed","timestamp":1694190172758,"scope":"notesnook-web","elapsed":346}
{"level":3,"message":"Sync status updated: synced","timestamp":1694190174434,"scope":"notesnook-web","elapsed":1676}
{"level":3,"message":"Access token requested","timestamp":1694190179863,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":109006}
{"level":3,"message":"Page visibility changed. Reconnecting SSE...","timestamp":1694190193777,"scope":"notesnook-web","elapsed":19343}
{"level":3,"message":"Sync status updated: syncing","timestamp":1694190193969,"scope":"notesnook-web","elapsed":192}
{"level":3,"message":"Auto sync requested","timestamp":1694190193972,"scope":"AutoSync","elapsed":24995}
{"level":3,"message":"Starting sync","timestamp":1694190193974,"extras":{"full":true,"force":false,"serverLastSynced":null},"scope":"Sync","elapsed":22556}
{"level":3,"message":"Initialized sync","timestamp":1694190194014,"extras":{"lastSynced":1694181965714,"oldLastSynced":1694181965714},"scope":"Sync","elapsed":40}
{"level":3,"message":"Data fetched","timestamp":1694190194305,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":291}
{"level":3,"message":"Uploading attachments...","timestamp":1694190194306,"extras":{"total":0},"scope":"Sync","elapsed":1}
{"level":3,"message":"Stopping sync","timestamp":1694190196432,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":0}
{"level":3,"message":"Sync status updated: completed","timestamp":1694190196446,"scope":"notesnook-web","elapsed":2477}
{"level":3,"message":"Access token requested","timestamp":1694190196480,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":25024}
{"level":3,"message":"refreshing app took 987.40ms","timestamp":1694190197435,"scope":"notesnook-web","elapsed":989}
{"level":3,"message":"Sync status updated: completed","timestamp":1694190197472,"scope":"notesnook-web","elapsed":37}
{"level":3,"message":"Sync status updated: synced","timestamp":1694190199460,"scope":"notesnook-web","elapsed":1988}
{"level":3,"message":"Sync status updated: failed","timestamp":1694192070291,"scope":"notesnook-web","elapsed":1870831}
{"level":1,"message":"Error: Connection disconnected with error 'Error: WebSocket closed with status code: 1006 (no reason given).'.\n    at Object.log (https://app.notesnook.com/assets/index-70bce852.js:375:127103)\n    at HttpConnection._stopConnection (https://app.notesnook.com/assets/index-70bce852.js:375:74295)\n    at transport.onclose (https://app.notesnook.com/assets/index-70bce852.js:375:71980)\n    at WebSocketTransport._close (https://app.notesnook.com/assets/index-70bce852.js:375:63211)\n    at at.onclose (https://app.notesnook.com/assets/index-70bce852.js:375:62107)","timestamp":1694192070293,"scope":"SignalR::SyncHub","elapsed":2948636}
{"level":3,"message":"Sync status updated: failed","timestamp":1694192070294,"scope":"notesnook-web","elapsed":3}
{"level":1,"message":"Error: WebSocket closed with status code: 1006 (no reason given).\n    at WebSocketTransport._close (https://app.notesnook.com/assets/index-70bce852.js:375:63219)\n    at at.onclose (https://app.notesnook.com/assets/index-70bce852.js:375:62107)","timestamp":1694192070295,"scope":"Sync","elapsed":1873863}
{"level":3,"message":"Page visibility changed. Reconnecting SSE...","timestamp":1694194118830,"scope":"notesnook-web","elapsed":2048536}
{"level":3,"message":"Sync status updated: syncing","timestamp":1694194119423,"scope":"notesnook-web","elapsed":593}
{"level":3,"message":"Auto sync requested","timestamp":1694194119426,"scope":"AutoSync","elapsed":3925454}
{"level":3,"message":"Starting sync","timestamp":1694194119434,"extras":{"full":true,"force":false,"serverLastSynced":null},"scope":"Sync","elapsed":2049139}
{"level":3,"message":"Access token requested","timestamp":1694194119469,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":4993282}
{"level":3,"message":"Refreshing access token","timestamp":1694194119470,"scope":"TokenManager","elapsed":1}
{"level":3,"message":"Access token requested","timestamp":1694194119480,"extras":{"accessToken":"A3BAB68E5E"},"scope":"TokenManager","elapsed":10}
{"level":3,"message":"Ignoring duplicate sync","timestamp":1694194120166,"extras":{"full":true,"force":false,"lastSynced":null,"syncing":true,"syncDisabled":false,"offline":false},"scope":"notesnook-web","elapsed":743}
{"level":3,"message":"Access token requested","timestamp":1694194120167,"extras":{"accessToken":"6C7076BFC9"},"scope":"TokenManager","elapsed":687}
{"level":3,"message":"Initialized sync","timestamp":1694194121075,"extras":{"lastSynced":1694181965714,"oldLastSynced":1694181965714},"scope":"Sync","elapsed":1641}
{"level":3,"message":"Data fetched","timestamp":1694194121406,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":331}
{"level":3,"message":"Uploading attachments...","timestamp":1694194121410,"extras":{"total":0},"scope":"Sync","elapsed":4}
{"level":3,"message":"Stopping sync","timestamp":1694194124266,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":0}
{"level":3,"message":"Sync status updated: completed","timestamp":1694194124288,"scope":"notesnook-web","elapsed":4122}
{"level":3,"message":"Access token requested","timestamp":1694194124307,"extras":{"accessToken":"6C7076BFC9"},"scope":"TokenManager","elapsed":3927827}
{"level":3,"message":"refreshing app took 843.80ms","timestamp":1694194125134,"scope":"notesnook-web","elapsed":846}
{"level":3,"message":"Sync status updated: completed","timestamp":1694194125161,"scope":"notesnook-web","elapsed":27}
{"level":3,"message":"Sync status updated: synced","timestamp":1694194127291,"scope":"notesnook-web","elapsed":2130}
{"level":3,"message":"Sync status updated: failed","timestamp":1694195749462,"scope":"notesnook-web","elapsed":1622171}
{"level":3,"message":"Sync status updated: failed","timestamp":1694195749467,"scope":"notesnook-web","elapsed":5}
{"level":1,"message":"Error: WebSocket closed with status code: 1006 (no reason given).\n    at WebSocketTransport._close (https://app.notesnook.com/assets/index-70bce852.js:375:63219)\n    at at.onclose (https://app.notesnook.com/assets/index-70bce852.js:375:62107)","timestamp":1694195749469,"scope":"Sync","elapsed":1625203}
{"level":3,"message":"Page visibility changed. Reconnecting SSE...","timestamp":1694196937113,"scope":"notesnook-web","elapsed":1187646}
{"level":3,"message":"Access token requested","timestamp":1694196937115,"extras":{"accessToken":"6C7076BFC9"},"scope":"TokenManager","elapsed":2812808}
{"level":3,"message":"Sync status updated: syncing","timestamp":1694196937680,"scope":"notesnook-web","elapsed":567}
{"level":3,"message":"Auto sync requested","timestamp":1694196937686,"scope":"AutoSync","elapsed":2818260}
{"level":3,"message":"Starting sync","timestamp":1694196937698,"extras":{"full":true,"force":false,"serverLastSynced":null},"scope":"Sync","elapsed":1188229}
{"level":3,"message":"Access token requested","timestamp":1694196937732,"extras":{"accessToken":"6C7076BFC9"},"scope":"TokenManager","elapsed":2817565}
{"level":3,"message":"Initialized sync","timestamp":1694196938574,"extras":{"lastSynced":1694181965714,"oldLastSynced":1694181965714},"scope":"Sync","elapsed":876}
{"level":3,"message":"Data fetched","timestamp":1694196938915,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":341}
{"level":3,"message":"Uploading attachments...","timestamp":1694196938918,"extras":{"total":0},"scope":"Sync","elapsed":3}
{"level":3,"message":"Stopping sync","timestamp":1694196941025,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":0}
{"level":3,"message":"Sync status updated: completed","timestamp":1694196941049,"scope":"notesnook-web","elapsed":3369}
{"level":3,"message":"Access token requested","timestamp":1694196941075,"extras":{"accessToken":"6C7076BFC9"},"scope":"TokenManager","elapsed":3960}
{"level":3,"message":"refreshing app took 1037.10ms","timestamp":1694196942088,"scope":"notesnook-web","elapsed":1039}
{"level":3,"message":"Sync status updated: completed","timestamp":1694196942124,"scope":"notesnook-web","elapsed":36}
{"level":3,"message":"Sync status updated: synced","timestamp":1694196944057,"scope":"notesnook-web","elapsed":1933}
{"level":3,"message":"Page visibility changed. Reconnecting SSE...","timestamp":1694197037823,"scope":"notesnook-web","elapsed":93766}
{"level":3,"message":"Sync status updated: syncing","timestamp":1694197038386,"scope":"notesnook-web","elapsed":563}
{"level":3,"message":"Auto sync requested","timestamp":1694197038390,"scope":"AutoSync","elapsed":100704}
{"level":3,"message":"Starting sync","timestamp":1694197038391,"extras":{"full":true,"force":false,"serverLastSynced":null},"scope":"Sync","elapsed":97366}
{"level":3,"message":"Initialized sync","timestamp":1694197038500,"extras":{"lastSynced":1694181965714,"oldLastSynced":1694181965714},"scope":"Sync","elapsed":109}
{"level":3,"message":"Data fetched","timestamp":1694197038809,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":309}
{"level":3,"message":"Uploading attachments...","timestamp":1694197038810,"extras":{"total":0},"scope":"Sync","elapsed":1}
{"level":3,"message":"Stopping sync","timestamp":1694197041555,"extras":{"lastSynced":1694181965714},"scope":"Sync","elapsed":0}
{"level":3,"message":"Sync status updated: completed","timestamp":1694197041598,"scope":"notesnook-web","elapsed":3212}
{"level":3,"message":"Access token requested","timestamp":1694197041633,"extras":{"accessToken":"6C7076BFC9"},"scope":"TokenManager","elapsed":100558}
```

**Comment by xd003** [comment]
I am guessing this has been fixed in v2. 6.3 https://github.com/streetwriters/notesnook/releases/tag/v2.6.3

**Comment by thecodrr** [comment]
This has been fixed in v2.6.3
